jsettlers.common/resources/buildings/big_livinghouse.xml

```diff
@@ -118,30 +116,26 @@
 	<blocked dx="-5" dy="-3" block="false" />
 	<blocked dx="-5" dy="-2" block="false" />
 	<blocked dx="-5" dy="-1" block="false" />
-	<blocked dx="-4" dy="1" block="false" />
```

Comment:
You will have less space for your bearers to move, especially when constructing the building. This is why there was that extra protected area. But we'll see how it works and if the jam issue is improved or not by this.